File: 7. Reverse Integer.js
var reverse = function(x) {
    var a = x.toString() 
    if (a[0] == '-') {
        a = a[0] + a.slice(1,).split('').reverse().join('')
        // console.log(a)
    }else{
        a = a.split("").reverse().join("");
    }
    a = parseInt(a)
    if ( -(2**31)<a && a<2**31){
        return a;
    }
    return 0;

};

File: 7. Reverse Integer.rb
def reverse(x)
    x = x.to_s
    if x[0] == '-' 
        x = (x[0]+x[1..].reverse).to_i 
    else
        x = x.reverse.to_i
    end
    (-2**31 < x && x <2**31-1)?(x):(0) 
        
end
